From be4b30069e35e084a9195bca8d3bed690f33fa6c Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 21 Aug 2018 11:48:31 +0100
Subject: [PATCH 09/10] klte-common: Update prefixes for audio system
 properties

 * All vendor-specific properties are required to have vendor prefixes

Change-Id: I562639e93b8eca4914473575c13da958224181f8
---
 system.prop | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/system.prop b/system.prop
index 9bc86ff..f30ff1d 100644
--- a/system.prop
+++ b/system.prop
@@ -3,19 +3,19 @@ dalvik.vm.dex2oat-swap=false
 
 # Audio
 af.fast_track_multiplier=1
-audio_hal.period_size=192
-audio.offload.video=true
-persist.audio.fluence.speaker=true
-persist.audio.fluence.voicecall=true
-persist.audio.fluence.voicerec=false
-ro.qc.sdk.audio.fluencetype=fluence
-use.voice.path.for.pcm.voip=false
-use.dedicated.device.for.voip=true
 audio.deep_buffer.media=true
+audio.offload.buffer.size.kb=32
 audio.offload.pcm.16bit.enable=true
 audio.offload.pcm.24bit.enable=true
-audio.offload.multiple.enabled=false
-audio.offload.buffer.size.kb=32
+audio.offload.video=true
+persist.vendor.audio.fluence.speaker=true
+persist.vendor.audio.fluence.voicecall=true
+persist.vendor.audio.fluence.voicerec=false
+ro.vendor.audio.sdk.fluencetype=fluence
+vendor.audio.offload.multiple.enabled=false
+vendor.audio_hal.period_size=192
+vendor.dedicated.device.for.voip=true
+vendor.voice.path.for.pcm.voip=false
 
 # Camera
 camera2.portability.force_api=1
-- 
2.17.1

